### DISCUSSION TITLE
Value in Rich Editor not being reset

### DISCUSSION THREAD

**Discussion by fatihdurmaz** [filamentphp/filament#13061] [Help]
Value in Rich Editor not being reset.
![Ekran görüntüsü 2024-06-02 215505](https://github.com/filamentphp/filament/assets/71706970/1b569f4b-97da-4968-a5f8-014be02a4b1a)

**Comment by leandrocfe** [comment]
Could you run `composer update -W` and try again?

**Reply by fatihdurmaz** [reply]
Yes, I try but it doesn't change.

**Comment by fatihdurmaz** [comment]
This is Bug Help!

**Reply by leandrocfe** [reply]
Are you able to create a minimal repo on Github to reproduce this issue?

**Reply by fatihdurmaz** [reply]
This error 

![Ekran Resmi 2024-06-03 13 14 02](https://github.com/filamentphp/filament/assets/71706970/b490901e-fc84-41c3-9fa6-07df638445a8)

**Comment by fatihdurmaz** [comment]
V3.2.76 working. This version later have a bug
